docs/infrastructure/requesting-infrastructure.md

```diff
@@ -0,0 +1,59 @@
+# Requesting infrastructure
+
+This page details the process for requesting new infrastructure. This aims to be
+as lightweight as possible, while capturing all the information that's needed.
+
+Unless otherwise noted, the way to request things is to contact the
+Infrastructure Team by starting a new conversation thread in the
+[`#infrastructure` channel][slack-infra] in Slack. Please ping
+`@infrastructure-committee` to ensure that your message is seen.
+
+In general you can expect a response within a couple of days (often much
+sooner), though note that the team may not always be able to act on your request
+for some time.
+
+Requests for new infrastructure must be made by (or confirmed by) members of a
+Team Committee.
```

Comment:
Should this be the budget holder on each committee? Is that worth it? (Do other people know who that is?)

Reply:
I believe other funding requests can be sent to the trustees by any committee member so any of them requesting infrastructure also seems reasonable

Reply:
Nevermind the above just seen the email which was different than the meeting discussion

Reply:
Yeah, that was my reasoning here. I think from our perspective sign-off from a committee member should suffice, especially to avoid that becoming too much of a bottleneck. I'm going to suggest we run with this approach for now and re-evaluate if it either becomes an issue or the trustees feel strongly otherwise (cc @dianedowling for visibility).

Reply:
Confirmed in this evening's all committees meeting -- @dianedowling is happy for team-level budget signoff to be by committee members (as long as the committee in question is ok with that).